bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/handler/BondBridgeHandler.java

```diff
@@ -242,23 +241,39 @@ public void forwardUpdateToThing(BPUPUpdate pushUpdate) {
      * Returns the Id of the bridge associated with the handler
      */
     public String getBridgeId() {
-        String serialNumber = config.serialNumber;
-        return serialNumber == null ? "" : serialNumber;
+        BondBridgeConfiguration configuration = config;
```

Comment:
You could also have config initialized with new BondBridgeConfiguration() so that no need to check if it is null everywhere. This is a common pattern in many bindings. 
That is just a suggestion, the code is also good as it is.

Reply:
Refactored the config init